terraform/roles/software_install/tasks/install_krew_plugins.yaml

```diff
@@ -0,0 +1,26 @@
+- name: "Install krew"
+  shell: |
```

Comment:
It would be nice to skip if krew is already present

Reply:
I'll do that when I work again in this repo. besides this one, the aliases are weird to be in this location as we could want other aliases as well, and it can be moved out to a role or subtask of its own.